service/src/main/java/org/cbioportal/service/util/ExpressionEnrichmentUtil.java

```diff
@@ -1,31 +1,21 @@
 package org.cbioportal.service.util;
 
 import java.math.BigDecimal;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 import java.util.Map.Entry;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
+import io.swagger.models.auth.In;
+import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.lang3.math.NumberUtils;
 import org.apache.commons.math3.stat.StatUtils;
 import org.apache.commons.math3.stat.descriptive.DescriptiveStatistics;
+import org.apache.commons.math3.stat.inference.ChiSquareTest;
 import org.apache.commons.math3.stat.inference.OneWayAnova;
 import org.apache.commons.math3.stat.inference.TestUtils;
-import org.cbioportal.model.EnrichmentType;
-import org.cbioportal.model.ExpressionEnrichment;
-import org.cbioportal.model.GenericAssayEnrichment;
-import org.cbioportal.model.GenericAssayMolecularAlteration;
-import org.cbioportal.model.GenomicEnrichment;
-import org.cbioportal.model.GroupStatistics;
-import org.cbioportal.model.MolecularAlteration;
-import org.cbioportal.model.MolecularProfile;
-import org.cbioportal.model.MolecularProfileCaseIdentifier;
-import org.cbioportal.model.MolecularProfileSamples;
-import org.cbioportal.model.Sample;
+import org.cbioportal.model.*;
```

Comment:
No * imports

Reply:
I have fixed these imports and push! Thanks a lot!